readme.md

```diff
@@ -56,6 +56,67 @@ RUNTIME_MODE       | no       | production               | Running mode (develop
 - Environment variables can also be set in the `./config.json` for development. If both provide a value for a specific property, the environment variable overrides the file. You can define your own `config.json` file and pass it to the application with the following command: `-c <config_file>`
 - The application can take any flags for [glog](https://github.com/golang/glog), which passes them straight into glog. The glog flag `--logtostderr` is set to true by default.
 
+### Dev Preview (Search Configurable Collection)
+
+Control resources that get collected from the Search-Collector by referencing an allow and deny list within search-collector-config configmap. Follow the following format in the sample template below:
+
+
+<pre><code>
+apiVersion: v1
+kind: ConfigMap
+metadata:
+ name: search-collector-config
+ namespace: Namespace where Search-Collector Pod is 
+data:
+ AllowedResources: |-
+   - apiGroups:
+       - "*"
+     resources:
+       - services
+       - pods
+   - apiGroups:
+       - admission.k8s.io
+       - authentication.k8s.io
+     resources:
+       - "*"
+ DeniedResources: |-
+   - apiGroups:
+       - "*"
+     resources:
+       - secrets
+   - apiGroups:
+       - admission.k8s.io
+     resources:
+       - policies
+       - iampolicies
+       - certificatepolicies
+</code></pre>
+
+Steps to create search-collector-config
+
+1. The <b>Namespace</b> value should be defined as the namespace where the Search-Collector pod lives.
+
+2. Use the <b>Name</b> `search-collector-config`
+
+3. Under <b>data</b> define `AllowedResources` and `DeniedResources` as key value pairs wrapped in a string block with `|-` to preserve linebreaks.
+
+    - The asterisk `"*"` represents <i>all</i>.
+
+    - Note that if you want to control resources for the current group then you should replace the `apiGroups` value with and empty string `""`. 
```

Comment:
Remove this statement, it's not accurate.

FYI, some resources don't have an apigroup, for those the apigroup is represented with an empty string. The resources that have an empty apigroup are the base kubernetes resources (Pod,ConfigMap,Secret, etc). All other extensions to the basic resources will have an apigroup.

Here's how to find the resources without a group. The resources at the top of this list only have version (v1) and no apigroup.
<img width="1145" alt="image" src="https://user-images.githubusercontent.com/4671325/160661460-7dde3f06-6cb6-42a9-89d0-fbb0f9967020.png">

Reply:
Ah, okay that makes sense. Thanks for clarifying!